<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pictureNote</class>
 <widget class="QMainWindow" name="pictureNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icos.qrc">
    <normaloff>:/new/prefix1/ICO/picture.ico</normaloff>:/new/prefix1/ICO/picture.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="5">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>474</width>
         <height>306</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="thePic">
          <property name="text">
           <string>图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QSpinBox" name="spinY">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="spinX">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>位置</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>水平距离</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>垂直距离</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>144</width>
         <height>192</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="theWater">
          <property name="text">
           <string>水印</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_opens"/>
    <addaction name="action_Save"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>水印(&amp;W)</string>
    </property>
    <addaction name="action_Insert"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于(&amp;H)</string>
    </property>
    <addaction name="action_Help"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_W"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/open.ico</normaloff>:/new/prefix1/ICO/open.ico</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/saveAs.ico</normaloff>:/new/prefix1/ICO/saveAs.ico</iconset>
   </property>
   <property name="text">
    <string>批量保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/clear.ico</normaloff>:/new/prefix1/ICO/clear.ico</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Insert">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/right_next.ico</normaloff>:/new/prefix1/ICO/right_next.ico</iconset>
   </property>
   <property name="text">
    <string>选择水印(&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/quit.ico</normaloff>:/new/prefix1/ICO/quit.ico</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/help2.ico</normaloff>:/new/prefix1/ICO/help2.ico</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_opens">
   <property name="icon">
    <iconset resource="icos.qrc">
     <normaloff>:/new/prefix1/ICO/ps.ico</normaloff>:/new/prefix1/ICO/ps.ico</iconset>
   </property>
   <property name="text">
    <string>批量打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>pictureNote</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
